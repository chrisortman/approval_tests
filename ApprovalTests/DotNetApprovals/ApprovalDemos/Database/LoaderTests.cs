using ApprovalDemos.Database.Loaders;
using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;

namespace ApprovalDemos.Data.Database
{
	[TestFixture]
	[UseReporter(typeof(BeyondCompareReporter))]
	public class LoaderTests
	{
 
		[Test]
			public void TestInsultLoader()
		{
			string conn = @"server=.\sqlexpress;database=Insults;trusted_connection=true";
			Approvals.Approve(new InsultLoaderShortAndSweet(7,5,conn));
		}
	}
}