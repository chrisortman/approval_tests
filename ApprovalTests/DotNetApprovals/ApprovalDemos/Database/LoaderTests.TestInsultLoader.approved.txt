
SELECT i.Insult, i.BurnLevel
		FROM Insults i
		WHERE 5 <= i.BurnLevel 
				AND i.NumberOfWords <= 7